page_objects/login_page.resource

```diff
@@ -1,47 +1,49 @@
 *** Settings ***
-Library        SeleniumLibrary
+Library    SeleniumLibrary
 
+Variables    ../env_conf.py
+Variables    ../constant/url_constants.py
+Variables    ../page_objects/web_locators/login_page_locators.py
+# Variables    ../test_data/default_data.yaml
 
-*** Variables ***
-${page_title}                 xpath://div[@class='page-title']/h1
-${email_txt}                  css:input#Email
-${password_txt}               css:input#Password
-${remember_checkbox}          css:input#RememberMe
-${login_btn}                  xpath://button[contains(text(),'Log in')]
-${login_error_msg}            xpath://div[@class='message-error validation-summary-errors']/ul/li
+Resource    ../utils/data_helper.resource
+Resource    ../browser/browser.resource
+Resource    ../page_objects/header_navigation.resource
+Resource    ../page_objects/register_page.resource
 
 *** Keywords ***
 Verify Page Title of Login Page
     Element Text Should Be    ${page_title}    Welcome, Please Sign In!
 
 Input Email To Login
     [Arguments]    ${login_email}
-    Input Text    ${email_txt}    ${login_email}
+    Wait Until Element Is Enabled    ${login_email_txt}
+    Input Text    ${login_email_txt}    ${login_email}
 
 Input Password To Login
     [Arguments]    ${login_pwd}
-    Input Text    ${password_txt}    ${login_pwd}
-
-Select Remember Checkbox
-    Select Checkbox    ${remember_checkbox}
+    Wait Until Element Is Enabled    ${login_password_txt}
+    Input Text    ${login_password_txt}    ${login_pwd}
 
 Click Login Button
+    Wait Until Element Is Enabled    ${login_btn}
     Click Button    ${login_btn}
 
 Login With
     [Arguments]    ${login_email}    ${login_pwd}
+    Verify Page Title of Login Page
     Input Email To Login    ${login_email}
     Input Password To Login    ${login_pwd}
     Click Login Button
 
+Login Successfully With
+    [Arguments]    ${login_email}    ${login_pwd}
+    Login With    ${login_email}    ${login_pwd}
+    Verify Log Out Link Appears
+
 Verify Error Message When Password Is Incorrect
     Element Text Should Be    ${login_error_msg}    The credentials provided are incorrect
 
-# Register New User Then Login
-#     Open The Register Page
-#     ${random_email}=    Generate Random String
-#     ${new_email}=    ${random_email}@mailinator.com
-#     Register With Required Data    ${default_firstName}    ${default_lastName}  
-#     ...    ${new_email}    ${default_pwd}    ${default_pwd}
-#     Click On Login Link
-#     Login With    ${new_email}    ${default_pwd}
+Open The Login Page
+    Open Browser    ${base_url}${login_path}    ${selected_browser}
+    Maximize Browser Window
```

Comment:
The 'Open The Home Page' keyword already includes code to maximize the browser, so please don't repeat that step again.

You only init browser and maximize it one time. It must be placed in browser.resource. However please not include any page link